packages/contracts-bedrock/scripts/generate-addresses.ts

```diff
@@ -0,0 +1,249 @@
+import glob from 'glob'
+import fs from 'fs'
+import path from 'path'
+import { z } from 'zod'
+import { Abi as AbiValidator, Address as AddressValidator } from 'abitype/zod'
+
+/**
+ * [zod](https://github.com/colinhacks/zod) validator for how the deployments json files
+ * are expected to be shaped
+ */
+const deploymentValidator = z.object({
+  address: AddressValidator,
+  abi: AbiValidator
+})
+type Deployment = z.infer<typeof deploymentValidator>
+type Address = z.infer<typeof AddressValidator>
+
+/**
+ * Predeployed contract addresses
+ * In future it would be nice to have a json file in contracts bedrock be generated as source of truth
```

Comment:
Think we can remove these comments now :D

Reply:
Good call lol

Reply:
Left a different comment so closing this one